tests/unit/test_messages_helpers.py

```diff
@@ -130,6 +131,31 @@ def test_getlogpath_exceeds_rotation_limit(test_log_dir, monkeypatch):
     assert present_logs == previous_fpaths[1:] + [new_fpath]
 
 
+def test_getlogpath_supports_missing_file_to_unlink(test_log_dir, monkeypatch):
+    """It's ok if the file to unlink was previously removed."""
```

Comment:
is this not more of a code comment? These docstrings are sometimes used as the summary of a test and this does not look like that.

Reply:
I use tests docstrings to explain what the test does (or try to).

Our infrastructure (pytest) shows `./E/F` in brief mode or the test function name when verbose.

Reply:
ok, this however does not sound like an explanation but more like a clarification